javascript/javascript1/week2/weather-wear.js

```diff
@@ -0,0 +1,29 @@
+function weatherChange(temperature){
```

Comment:
A function name should usually be inspired by what the function returns which in this case it "what to wear".